src/applications/personalization/rated-disabilities/components/RatedDisabilityTable.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SortableTable from '@department-of-veterans-affairs/formation-react/SortableTable';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import moment from 'moment';
+
+class RatedDisabilityTable extends React.Component {
+  static propTypes = {
+    fetchRatedDisabilities: PropTypes.func.isRequired,
+    ratedDisabilities: PropTypes.shape({
+      ratedDisabilities: PropTypes.array,
+    }),
+  };
+
+  componentDidMount() {
+    this.props.fetchRatedDisabilities();
+  }
+
+  noDisabilityRatingContent = errorCode => {
+    let headline;
+    let content;
+    let status;
+    switch (errorCode) {
+      case '500':
+        headline = 'Rated disabilities error';
+        status = 'error';
+        content = (
+          <>
+            <p>
+              We're sorry. An error occurred when accessing your disability
+              rating information.
+            </p>
+            <h4>What you can do</h4>
+            <p>
+              Sign out of VA.gov, then log back in to try this page again. If
+              the error continues, please call the VA.gov Help Desk at
+              1-855-574-7286 (TTY:1-800-829-4833). We're here Monday-Friday,
+              8:00 a.m. - 8:00 p.m. (ET).
+            </p>
+          </>
+        );
+        break;
+      default:
+        headline = 'No disability rating found';
+        status = 'info';
+        content = (
+          <>
+            <p>
+              We sorry. We can't find a disability rating matched with the name,
+              date of birth, and social secuity number you provided in our
+              Veteran records.
+            </p>
+            <h4>What you can do</h4>
+            <p>
+              If you feel your information is correct, please call the VA.gov
+              1-855-574-7286. We're here Monday through Friday, 8:00 a.m. to
+              8:00 p.m. (ET).
+            </p>
+          </>
+        );
+        break;
```

Comment:
Abstracting these error messages into their a stateless React component sometimes helps with unit tests. For example, your unit test can just check for the `<NoDisabilityRatingFound/>` component instead of a string value.